scripts/remove_duplicate_preprint_logs.py

```diff
@@ -0,0 +1,62 @@
+import sys
+import logging
+
+from modularodm import Q
+
+from framework.transactions.context import TokuTransaction
+from scripts import utils as script_utils
+from website.app import init_app
+from website.project.model import Node, NodeLog
+
+
+logger = logging.getLogger(__name__)
+
+
+# This is where all your migration log will go
+def do_migration():
+    dupe_nodes = [n for n in Node.find(Q('_id', 'in', list(set([l.node._id for l in NodeLog.find(Q('action', 'eq', 'preprint_license_updated'))])))) if NodeLog.find(Q('action', 'eq', 'preprint_license_updated') & Q('node', 'eq', n._id)).count() > 1]
+    logger.info('Found {} nodes with multiple preprint_license_updated logs'.format(len(dupe_nodes)))
+
+    for node in dupe_nodes:
+        preprint_license_updated_logs = [log for log in node.logs if log.action == 'preprint_license_updated']
+
+        log = preprint_license_updated_logs.pop()
+        while(preprint_license_updated_logs):
+            next_log = preprint_license_updated_logs.pop()
+            if (log.date - next_log.date).seconds < 1:
+                logger.info('Removing duplicate preprint_license_updated log with ID {} from node {}'.format(log._id, node._id))
+                log.should_hide = True
+                log.save()
+
```

Comment:
Could you increase logging here? Important things to log tend to be what you're doing and why -- the time delta and logs that you've found but don't migrate would be good to know.